docs/en/observability/aws-firehose/aws-firehose-setup.asciidoc

```diff
@@ -0,0 +1,203 @@
+If you are using an API key with “Restrict privileges”, be sure to review the https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html#privileges-list-indices[Indices privileges] to provide at least  "auto_configure" & "write" permissions for the indices you will be using with this delivery stream.
++
+[discrete]
+[[aws-firehose-config-destination-settings-content-encoding]]
+// <h6>
+==== Content encoding
+We recommend leaving *Content encoding* set to *GZIP* for improved network efficiency. 
++
+[discrete]
+[[aws-firehose-config-destination-settings-retry-duration]]
+// <h6>
+==== Retry duration
+*Retry duration* determines how long Firehose continues retrying the request in the event of an error.
+A duration of 60-300s should be suitable for most use cases.
++
+[discrete]
+[[aws-firehose-config-destination-settings-parameters]]
+// <h6>
+==== Parameters
+
+* When using Elastic integrations, setting the *`es_datastream_name`* parameter is required.
++ 
+Elastic integrations use data streams with specific naming conventions, and Firehose records need to be routed to the relevant data stream to make use of preconfigured index mappings, ingest pipelines and dashboards.
++
+_A separate Firehose delivery stream is required for each log type in AWS to make use of Elastic integrations._
++ 
+The following is a list of common AWS log types and the *`es_datastream_name`* value that needs to be set to route the logs to the correct integration.
++
+[cols="1,3"]
+|===
+| AWS log type | *`es_datastream_name`* value
+
+| https://docs.elastic.co/en/integrations/aws/cloudfront[Cloudfront]
+| `logs-aws.cloudfront_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/ec2[EC2 (via Cloudwatch)]
+| `logs-aws.ec2_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/elb[ELB]
+| `logs-aws.elb_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/firewall[Network firewall]
+| `logs-aws.firewall_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/route53[Route 53 public DNS queries]
+| `logs-aws.route53_public_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/route53[Route 53 resolver queries]
+| `logs-aws.route53_resolver_logs-default`
+
+| https://docs.elastic.co/en/integrations/aws/s3[S3 server access]
+| `logs-aws.s3access-default`
+
+| https://docs.elastic.co/en/integrations/aws/vpcflow[VPC Flow Logs]
+| `logs-aws.vpcflow-default`
+
+| https://docs.elastic.co/en/integrations/aws/waf[WAF]
+| `logs-aws.waf-default`
+
+|===
++
+As per the https://www.elastic.co/blog/an-introduction-to-the-elastic-data-stream-naming-scheme[data stream naming conventions], the "namespace" is a user-configurable arbitrary grouping, and can be changed from `default` to fit your use case. For example, you may want to organize WAF Logs per environment into `logs-aws.waf-production` and `logs-aws.waf-qa` data streams for more granular control over rollover, retention, and security permissions.
```

Comment:
I'd rather link to the documentation about data stream naming since that will continue to be updated:

```suggestion
As per the {fleet-guide}/data-streams.html[data stream naming conventions], the "namespace" is a user-configurable arbitrary grouping and can be changed from `default` to fit your use case. For example, you may want to organize WAF Logs per environment into `logs-aws.waf-production` and `logs-aws.waf-qa` data streams for more granular control over rollover, retention, and security permissions.
```